specs/MultiProfile.md

```diff
@@ -363,16 +394,42 @@ interface ICoreWebView2Profile2 : ICoreWebView2Profile {
   [propget] HRESULT CookieManager([out, retval] ICoreWebView2CookieManager** cookieManager);
 }
 
-[uuid(1c1ae2cc-d5c2-ffe3-d3e7-7857035d23b7), object, pointer_default(unique)]
-interface ICoreWebView2Profile3 : ICoreWebView2Profile2 {
-  /// All webviews on this profile will be closed, and the profile will be marked for deletion.
-  /// After the Delete() call completes, The render process of webviews on this profile will
-  /// asynchronously exit with the reason:`COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED`.
-  /// See 'COREWEBVIEW2_PROCESS_FAILED_REASON::COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED'
-  /// for more details. The profile directory on disk will be actually deleted when the browser
-  /// process exits. Webview2 creation will fail with the HRESULT is ERROR_INVALID_STATE(0x8007139FL)
-  /// if you create it with the same name as a profile that is being deleted.
-  HRESULT Delete(); 
+[uuid(2765B8BD-7C57-4B76-B8CC-1EC940FF92CC), object, pointer_default(unique)]
+interface ICoreWebView2StagingProfile7 : IUnknown {
+  /// After the API is called, the profile will be marked for deletion. The
+  /// local profile's directory will be deleted at browser process exit. If it
+  /// fails to delete, because something else is holding the files open,
+  /// WebView2 will try to delete again during all future at next browser
+  /// process starts until successful.
+  /// The corresponding CoreWebView2s will be closed and the 
+  /// CoreWebView2Profile.Deleted event will be raised. See
+  /// `CoreWebView2Profile.Deleted` for more information.
+  /// If you try to create a new profile with the same name as an existing
+  /// profile that has been marked as deleted but hasn't yet been deleted,
+  /// profile creation will fail with HRESULT_FROM_WIN32(ERROR_INVALID_STATE).
+  HRESULT Delete();
+
+  /// Add an event handler for the `Deleted` event. The `Deleted` event is
+  /// raised when the profile is marked for deletion. When this event is
+  /// raised, the CoreWebView2Profile and its corresponding CoreWebView2s are
+  /// closed, and cannot be used anymore.
```

Comment:
"are closed" is ambiguous. Does it mean 'have been closed' or 'will be closed'? I.e., does this fire before or after the close?

Reply:
This event fire after close and will change to `'have been closed`.